src/_designs/screens/dashboard/index.vue

```diff
@@ -3,18 +3,30 @@
     <PageHeader title="Dashboard">
       <BreadCrumbs />
     </PageHeader>
+    <h1>Generations</h1>
     <Generations v-if="generations">
       <Generation
         v-for="(generation, number) in generations"
         :key="number"
         :data="generation"
       />
     </Generations>
+    <br>
+    <h1>Transactions</h1>
```

Comment:
Both the titles(Generation and Transaction) can be inside the v-if. Presently, when the generations, the transactions haven't yet loaded, the two headings stick together.

Reply:
updated